### DISCUSSION TITLE
Livewire not send updated value

### DISCUSSION THREAD

**Discussion by tarun-pixeleyez** [livewire/livewire#8223] [1. Help]
Hello,
I am creating chat application using Livewire or Laravel Revert. When i dispatch event on send message the Array of message was updated but not fetch and see in chat and also UI was crashed.

here the my component.

`class Chatbox extends Component
{
    public $message = '';
    public $receiver_id = 2;
    public $messages = [];

    protected $chatRepository;

    public function boot(ChatRepository $chat)
    {
        $this->chatRepository = $chat;
        $this->loadMessages();
    }

    public function loadMessages()
    {
        $this->messages = $this->chatRepository->getMessagesForUser(auth()->user()->id);
    }

    #[On('echo:chatroom,.message.sent')]
    public function onMessageSent($event)
    {
        $this->messages[] = $event['message'];
    }

    public function addMessage()
    {
        $data = [
            'from_user' => auth()->user()->id,
            'to_user' => $this->receiver_id,
            'message' => $this->message
        ];

        $this->chatRepository->create($data);

        $this->message = '';
    }

    public function render()
    {
        return view('livewire.chatbox');
    }
}`

and my html rendring
`<div class="chat-body p-3" tabindex="1" data-simplebar>
        @foreach ($messages as $message)
            <h6>{{ $message['message'] }}</h6>
        @endforeach
    </div>`
    
    Please heml me with that.
    
![image](https://github.com/livewire/livewire/assets/159905617/212fdcfc-a9fd-4e42-9db4-93e60029c4f1)
![Uploading image.png…]()

**Comment by sanderdewijs** [comment]
Have you tried using the mount() method instead of boot() ?

**Reply by tarun-pixeleyez** [reply]
Yes I have tried but its not works.
